app/scripts/controllers/ticketSearch.ctrl.js

```diff
@@ -98,7 +98,14 @@ angular.module('yourStlCourts').controller('ticketSearchCtrl', function (Citatio
       if(result.citations.length > 0) {
         $state.go('citationInfo', {citations: result.citations});
       } else {
-        toaster.pop('error', 'Oh no! There were no tickets matching your criteria');
+        var homeLink = '<a href="/#/home"><u>clicking here</u></a>'
```

Comment:
Please avoid hard-coding ```/#/``` in the code since that will be gone soon. In this case, since you're redirecting to the homepage, just use ```href="/"```